win32ss/user/ntuser/class.c

```diff
@@ -1865,11 +1867,37 @@ IntSetClassMenuName(IN PCLS Class,
     return Ret;
 }
 
+static inline
+ULONG_PTR
+IntGetSetClassLongPtr(PCLS Class, ULONG Index, ULONG_PTR NewValue, ULONG Size)
+{
+    PVOID Address = (PUCHAR)(&Class[1]) + Index;
+    ULONG_PTR OldValue;
+
+#ifdef _WIN64
+    if (Size == sizeof(LONG))
```

Comment:
```suggestion
    if (Size == sizeof(ULONG))
```
(that's the same, but for code self consistency)